src/api/serviceConfig.ts

```diff
@@ -1,7 +1,9 @@
+import { GoDaddy } from '../constants/providers'
+
 const serviceConfig: unknown = {
-  dns_gd: {
-    name: 'Godaddy',
-    keys: ['GD_Key', 'GD_Secret']
+  [GoDaddy.DNS_API]: {
+    name: GoDaddy.NAME,
+    keys: [GoDaddy.PRIMARY_KEY, GoDaddy.SECONDARY_KEY]
```

Comment:
I don't see this change having any benefits, its actually makes things more confusing right?

Reply:
Really ?
The benefit is to have only one place to set up new provider
At `src/constants/providers.ts`
```js
export const GoDaddy = {
  NAME: 'GoDaddy',
  DNS_API: 'dns_gd',
  PRIMARY_KEY: 'GD_Key',
  SECONDARY_KEY: 'GD_Secret',
  SERVICE: 'https://api.godaddy.com'
} as ProviderInfo
```
Maybe  I can improve it with a function that will read `src/constants/providers.ts` and adding them within `src/api/serviceConfig.ts` ?